package.json

```diff
@@ -38,8 +38,8 @@
     "brunch": "./bin/brunch"
   },
   "scripts": {
-    "test": "eslint lib test && ava",
-    "test:coverage": "nyc ava && nyc report --reporter=html"
+    "test": "eslint lib test && ava -s",
+    "test:coverage": "nyc ava -s && nyc report --reporter=html"
```

Comment:
Since now we have test/build.js and test/watch.js, by default these files will run serially, but we need to run files sequentially to prevent that both write to the tmp test dir at the same time.